continuous_integration/environment-3.10.yaml

```
@@ -38,7 +38,7 @@ dependencies:
   - sortedcollections
   - tblib
   - toolz
-  - tornado<6.2
+  - tornado >=6.2
```

This seems like the version restriction we want, given what's specified in `requirements.txt`

```suggestion
  - tornado >=6.0.3
```

---

if the restriction doesn't require >=6.2 in the environment then it follows what's currently released for the distributed package

---

> I think the problem we're having here is that all python versions are now testing against 6.2 but we're not testing against 6.0.3 anymore, are we?

Yeah, that's what I was wanting to avoid. 

> I'm fine with this for now if we add <6.2 to https://github.com/dask/distributed/pull/7285

Yep, I think the mindeps build will be the long term answer for testing against our lowest supported minimum version. 

FWIW I didn't mean for this comment to be majorly blocking. @graingert if you and @fjetter are happy with things as they are, that's fine by me.